### ISSUE TITLE
Adaptive icon

### ISSUE THREAD

**Issue by afollestad** [naman14/TimberX#31]
Self explanatory

**Comment by hutriteguh** [comment]
hey @afollestad I just made this icon, wdyt?

![image](https://user-images.githubusercontent.com/46064613/52529809-6ef67500-2d2b-11e9-8598-84a542560bc0.png)

**Comment by afollestad** [comment]
It's up to the app owner, @naman14

**Comment by afollestad** [comment]
In the meantime, I have https://github.com/naman14/TimberX/pull/46 as well

**Comment by hutriteguh** [comment]
About the adaptive icon, I think we only need to provide the icon in a square background since most of android launchers already have adaptive icon feature.

**Comment by afollestad** [comment]
You provide a background and a foreground which are wrapped in an `adaptive-icon` drawable.

In most cases the background is just a solid color, while the foreground is a Vector Drawable.

**Comment by hutriteguh** [comment]
I can provide the .svg file and Android Studio will convert it to vectordrawabel

**Comment by afollestad** [comment]
Yep - there's a chance Android Studio will have trouble converting because of certain attributes like evenOdd fill type which isn't supported

**Comment by hutriteguh** [comment]
got it, I'll wait for app owner if he agree to use this icon.

**Comment by hutriteguh** [comment]
btw, i jst try to convert the .svg to vectordrawable and worked perfectly without any bug

![image](https://user-images.githubusercontent.com/46064613/52530021-78cea700-2d30-11e9-9579-5e7ef89e6842.png)

**Comment by hutriteguh** [comment]
@afollested what about the new logo?
